src/main/java/uk/gov/hmcts/ccd/domain/service/aggregated/DefaultFindSearchInputOperation.java

```diff
@@ -47,16 +52,17 @@ private SearchInput toSearchInput(final SearchInputField in, final CaseType case
         result.setOrder(in.getDisplayOrder());
         final Field field =new Field();
         field.setId(in.getCaseFieldId());
-        field.setType(getFieldType(in.getCaseFieldId(), caseType));
+        CaseField caseField = getFieldType(in.getCaseFieldId(), caseType);
+        field.setType(caseField.getFieldType());
+        field.setMetadata(caseField.isMetadata());
         result.setField(field);
         return result;
     }
 
-    private FieldType getFieldType(final String fieldId, final CaseType caseType){
+    private CaseField getFieldType(final String fieldId, final CaseType caseType) {
```

Comment:
Method should be renamed as it differs from return type now.

Reply:
good catch. Renamed now